frontend/src/pages/profile/styles.module.scss

```diff
@@ -0,0 +1,37 @@
+.profile {
+  width: 1591px;
```

Comment:
What would be on screens narrower than `1591px`?

Reply:
Horizontal scroll will appear in this case. I change it to 
.profile {
  max-width: 1591px;
  width: 100%;
  margin: 0 auto;
}